FrontEnd/Hopsy/src/screens/ProfileScreen.js

```diff
@@ -29,19 +29,19 @@ export default class ProfileScreen extends React.Component {
           username: GLOBAL.user,
         })
     })  
-    .then((response) => response.text())
-      .then((responseJson) => {
-        this.setState({
-          fullname: responseJson,
-        }, function() {
-        })
-      })
+    .then(response => {
+      this.setState({
+        fullname: JSON.parse(response),
+      }, function() {
+      });
+    })
     .catch((error) =>{
       console.error(error);
     });
-
   }
 
+
+  
   componentDidMount() {
```

Comment:
This will have to get changed to allow a user to actually log out. Component did mount will only call the fetch a single time when the app is opened. So if a user logs out, the previous person's name will still be rendered on the screen

Reply:
I changed how logging out functioned to prevent this issue.  After changes componentDidMount() functions as desired.  Issue resolved when #158 was closed.